src/cc_catalog_airflow/dags/util/loader/provider_details.py

```diff
@@ -22,6 +22,15 @@
 SCIENCE_DEFAULT_PROVIDER = 'sciencemuseum'
 STATENS_DEFAULT_PROVIDER = 'statensmuseum'
 WALTERS_DEFAULT_PROVIDER = 'waltersartmuseum'
+FINNISH_DEFAULT_PROVIDER = 'finnishmuseums'
+
+# Finnish parameters
+FINNISH_SUB_PROVIDERS = {
+    'national_museum_of_finland': '0/Suomen kansallismuseo/',
+    'finnish_heritage_agency': '0/Museovirasto/',
+    'satakunnan_museum': '0/SATMUSEO/',
+    'military_museum': '0/SA-kuva/'
```

Comment:
please add `finnish` or `finland` in these source names for clarity in the DB.  Thank you!